### PR TITLE
JUP-71 Backend Changes

### PR DESCRIPTION
**Pull request by BK2004**
- Added officers table for listed officers
- Removed title field from userMetadataToClubs and all references to it

### CODE REVIEW

File: src/server/db/schema/officers.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    clubId: text('club_id')
+      .notNull()
+      .references(() => club.id),
+    name: text('name').notNull(),
+    position: text('position').notNull(),
+    image: text('image').default('/nebula-logo.png').notNull(),
+    isPresident: boolean('is_president').default(false).notNull(),
+  },
+  (table) => ({
+    pk: primaryKey(table.clubId, table.id),
```
Comment (RubenOlano) [inline src/server/db/schema/officers.ts:18]: Why are we doing composite key here?

File: src/server/db/schema/officers.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { relations } from 'drizzle-orm';
+import { pgTable, text, boolean, primaryKey } from 'drizzle-orm/pg-core';
+import { club } from './club';
+
+export const officers = pgTable(
+  'officers',
+  {
+    id: text('id').notNull(),
```
Comment (RubenOlano) [inline src/server/db/schema/officers.ts:8]: Need to set some default value.
Maybe `uuid`?

**Review by RubenOlano — COMMENTED** [review]
Couple of questions here let me know if I'm just being dumb

File: src/server/db/schema/officers.ts (hunk 1 above)
Comment (BK2004) [reply src/server/db/schema/officers.ts:18]: Now that I'm looking at it, there really isn't a reason for a composite key. I'll get rid of that.

Comment (RubenOlano) [reply src/server/db/schema/officers.ts:18]: Sounds good. I was thinking that maybe it was supposed to be linked with the `userId` but it's not a column in the table so prob just leave it out

File: src/server/db/schema/officers.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { relations } from 'drizzle-orm';
+import { pgTable, text, boolean, uuid } from 'drizzle-orm/pg-core';
+import { club } from './club';
+
+export const officers = pgTable('officers', {
+  id: uuid('id').defaultRandom().notNull().primaryKey(),
```
Comment (RubenOlano) [inline src/server/db/schema/officers.ts:6]: Unnecessary `notNull`

**Review by RubenOlano — CHANGES_REQUESTED** [review]
One final nitpick

**Review by RubenOlano — APPROVED** [review]
LGTM

**Review by nl32 — APPROVED** [review]
LGTM

**Pull request merged**